ui/.env.production

```diff
@@ -1,2 +1,2 @@
-PUBLIC_URL="/MAAS"
+PUBLIC_URL="/MAAS/r/"
```

Comment:
This is required so that the assets built by React get the correct urls. This can be removed once the backend serves the UI at `/MAAS/`